packages/gatsby-plugin-image/README.md

```diff
@@ -226,34 +226,44 @@ export const query = graphql`
 
 ### Upgrading from the gatsby-image@2
 
-We will be releasing a codemod to automatically update your queries and imports. In the meantime, you can use the compat layer to make the transformation easier. This will be removed when we leave beta, but for now, it allows you to try the component with your existing queries.
+We've included a codemod to help you migrate to the new `gatsby-plugin-image` API.
 
-```jsx
-import React from "react"
-import { graphql } from "gatsby"
-// Note the different import
-import { GatsbyImage } from "gatsby-plugin-image/compat"
-
-export default ({ data }) => (
-  <div>
-    <h1>Hello gatsby-image</h1>
-    <GatsbyImage fixed={data.file.childImageSharp.fixed} />
-  </div>
-)
+1. Install JSCodeshift as a global module
 
-export const query = graphql`
-  query {
-    file(relativePath: { eq: "blog/avatars/kyle-mathews.jpeg" }) {
-      childImageSharp {
-        fixed(width: 125, height: 125) {
-          ...GatsbyImageSharpFixed
-        }
-      }
-    }
-  }
-`
+```shell
+npm install -g jscodeshift
+```
+
+2. Install this package
+
+```shell
+npm install gatsby-plugin-image
+```
+
+3. Add `gatsby-plugin-image` to your `gatsby-config.js` file.
+
+4. Make sure `gatsby-transformer-sharp` and `gatsby-plugin-sharp` are updated to the latest versions.
+
+5. Run a transform from this package on your project. Note that the paths are relative to the root of your project.
+
+```shell
+jscodeshift -t node_modules/gatsby-plugin-image/src/transforms/gatsby-plugin-image.js ./src -s
```

Comment:
This file won't be included in the npm package

Reply:
Ya, realizing the tests we've run bypassed the official build process. Can align on location and then fix.